core/stats.h

```diff
@@ -207,11 +207,11 @@ extern timestamp_t kstat_ignore_context_switch;
 
 #    ifdef DEBUG
 /* avoid recursion in logging KSTAT - do not checkin uses */
-#        define KSTAT_DUMP_STACK(kstack)                                       \
-            if ((kstack)->node[(kstack)->depth - 1].var !=                     \
-                &cur_dcontext->thread_kstats->vars_kstats.logging) {           \
-                LOG(THREAD_GET, LOG_STATS, 1, "ks %s:%d\n"__FILE__, __LINE__); \
-                kstats_dump_stack(cur_dcontext);                               \
+#        define KSTAT_DUMP_STACK(kstack)                                        \
```

Comment:
Why did this change?

Reply:
Because C++ won't accept `"string"__FILE__` without a space between them. If I stop using C++ I can revert this part of the change.